docs/Build.md

```diff
@@ -90,19 +91,17 @@ The final step is to configure the `hie` client to use a custom `hie-wrapper` sc
 The `install.hs` script performs some checks to ensure that a correct installation is possible and provide meaningful error messages for known issues.
 
 * `stack` needs to be up-to-date. Version `1.9.3` is required
+* `cabal` needs to be up-to-date. Version `3.0.0.0` is required for windows systems and `2.4.1.0` for other ones.
```

Comment:
We are not supporting cabal 3.0.0.0 before #1126 has landed, right?

Reply:
We support it for *build* hie but no for *use* hie. It is not consistent and i would stick to `2.4.1.0` but it can be used to build hie in windows

Reply:
That is troublesome.

Reply:
Well, hie-bios are going to land really soon 😉
 i'll update the phrase to make ir clear.

Reply:
@fendor i've rephrased the requirements for cabal in the docs